code/modules/projectiles/guns/ballistic/revolver.dm

```diff
@@ -343,3 +343,16 @@
 		new /obj/item/stack/cable_coil(get_turf(src), 10)
 		slung = 0
 		update_icon()
+
+/obj/item/weapon/gun/ballistic/revolver/reverse //Fires directly at its user... unless the user is a clown, of course.
+	clumsy_check = 0
+
+/obj/item/weapon/gun/ballistic/revolver/reverse/can_trigger_gun(mob/living/user)
+	if((user.disabilities & CLUMSY) || user.mind && user.mind.assigned_role == "Clown")
```

Comment:
put `user.mind && user.mind.assigned_role == "Clown"` in parentheses so that byond doesn't maybe fuck up